.gitignore

```diff
@@ -7,4 +7,5 @@
 *.out
 actions/__tmp
 vendor/
-actions/__tmp
\ No newline at end of file
+api/tmp/
+api/models.ts*
```

Comment:
could we also add the model backups here?

Reply:
that's what the _*_ is for ;)

Reply:
shouldn't those files be removed from this pr then?